l10n_nl_postcode/models/res_partner.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2016-2018 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, api, models
+
+_logger = logging.getLogger(__name__)
+try:
+    from stdnum.nl import postcode
+except ImportError:
+    _logger.debug('Cannot import stdnum.nl.postcode.')
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    @api.multi
+    def _l10n_nl_postcode_get_warning(self):
+        self.ensure_one()
+        msg = _('The Postcode you entered (%s) is not valid.')
+        warning = {
+            'title': _('Warning!'),
+            'message': msg % self.zip,
+        }
+        return warning
+
+    @api.multi
+    def _l10n_nl_postcode_check_country(self):
+        self.ensure_one()
+        country = self.country_id
+        if not country or country != self.env.ref('base.nl'):
```

Comment:
The first part of the `if` condition is not useful, as it's implicitely included in the second part. Moreover, you could also write the whole check more compactly:

`if self.country_id != self.env.ref('base.nl'):`

This should largely suffice

Reply:
I would keep it as it is. It's meant to handle the case that 'base.nl' is not present in the system for some reasons.

Reply:
Fine.